### PR TITLE
Make background job time insensitive

### PR DESCRIPTION
**Pull request by nickvergessen**
Ref https://github.com/nextcloud/server/pull/30945

### CODE REVIEW

**Review by miaulalala — APPROVED** [review]

File: tests/AppInfo/ApplicationTest.php [hunk 1]
```diff
@@ -80,44 +80,9 @@ public function testContainerAppName(): void {
...
-			[APIv1Controller::class],
-			[APIv1Controller::class, Controller::class],
-			[APIv1Controller::class, OCSController::class],
-			[APIv2Controller::class],
-			[APIv2Controller::class, Controller::class],
-			[APIv2Controller::class, OCSController::class],
-			[FeedController::class],
-			[FeedController::class, Controller::class],
-			[SettingsController::class, SettingsController::class],
-			[SettingsController::class, Controller::class],
```
Comment (artonge) [inline tests/AppInfo/ApplicationTest.php:120]: How come we don't need that anymore?

**Review by artonge — APPROVED** [review]

File: tests/AppInfo/ApplicationTest.php (hunk 1 above)
Comment (nickvergessen) [reply tests/AppInfo/ApplicationTest.php:120]: It uses normal DI, so should be fine.
The entries that I kept uses custom services defined in application class.
But we can also keep them. then let me fix the TimedJob one

Comment (artonge) [reply tests/AppInfo/ApplicationTest.php]: If this is testing the DI logic, then it's unrelated and I am fine with not keeping it :)

**Pull request merged**